yaml/buildings.yaml

```diff
@@ -18,6 +18,12 @@ globals:
       roof_orientation: {col: "roof:orientation"}
       addr_housenumber: {col: "addr:housenumber"}
       addr_street: {col: "addr:street"}
+  - &building_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail(building)
+  - &building_part_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail("building:part")
```

Comment:
What's the overlap between the popular `building:part` values and `building` values? Are there any popular `building:part` values we're missing from the list in `mz_building_kind_detail()`?